valohai_yaml/schema/step.yaml

```diff
@@ -50,3 +50,18 @@ properties:
   environment:
     type: string
     description: Default execution environment (ID or substring of environment name)
+  time-limit:
+    oneOf:
+      - type: integer
+      - type: string
+    description: >
+      The time limit for the step (in seconds or as a string (e.g. "1h 30m 5s")).
```

Comment:
Nit: could drop the () around "e.g. ..." as the double parenthesis look a bit off?

Reply:
De-nested! 💪